src/commands/game_commands/lookup.ts

```diff
@@ -114,6 +116,7 @@ async function lookupByYoutubeID(
     let artistName: string;
     const songAliases: string[] = [];
     const artistAliases: string[] = [];
+    const tags: string = getEmojisFromSongTags(daisukiEntry);
```

Comment:
nit: variable name needed to be updated
nit: this variable is only used in one place, can we just call `getEmojisFromSongTags()` directly where it's used?